COMMUNITY_MEMBERSHIP.md

```diff
@@ -134,7 +134,7 @@ The process for becoming a maintainer should be defined in the project or subpro
     meetings)
   - Identifying subtle or complex issues in designs and implementation PRs
 - Directly contributed to the subproject through implementation and / or review
-- Aligning with the overall project goals, specifications, and design principles defined by Technical Committee (TC). Bringing general questions and requests to the discussions as part of specifications project.
+- Aligning with the overall project goals, specifications, and design principles defined by Technical Committee (TC) (NEEDS CLARIFICATION). Bringing general questions and requests to the discussions as part of specifications project.
```

Comment:
similar to the GC, where is the TC defined and do we need it?

Reply:
Please see the comments below.